Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1692,7 +1692,12 @@ def delete_context_table_records(client: Client, args: Dict) -> Tuple[Any, Dict[
     session_id = args.get('session_id')
     records = argToList(args.get('records'))
 
-    record_updates_raw_data = client.delete_context_table_records_request(context_table_name, records, session_id)
+    records_raw_data = client.list_context_table_records_request(context_table_name, 10000, 1)
+    all_records = records_raw_data.get('records', [])
+    filtered = list(filter(lambda record: (record['key'] in records), all_records))
+    ids = [record['id'] for record in filtered]
```

Comment:
this is more readable and has a better performance.
```suggestion
    ids = [record['id'] for record in filtered if record['key'] in records]
```